File: level1/1.13.c

int testResult(int n,int arr[]){
    int min = 100;
    int max = 0;
    int sum;

    for(int i; i < n; i++) {
        if (arr[i] > max) max = arr[i];
        if (arr[i] < min) min = arr[i];
        sum =+ arr[1];
    };

    int median;

    if ((median % 1) != 0 ) median = ; // тоді вивести медіану дробом
    int median = sum / n;

    int above_average = 0;
    for(int i; i < n; i++) {
        if (arr[i] > median) above_average =+ 1;
    };
};

File: level1/1.13.py
N = int(input())
arr = list(map(int, input().split()))


print(f"Min: {min(arr)}")
print(f"Max: {max(arr)}")
print(f"Median: {sorted(arr)[len(arr)//2] if len(arr) % 2 == 1 else (sorted(arr)[len(arr)//2][0] + sorted(arr)[len(arr)//2][1]) / 2}")
print(f"Above average: {len([1 for num in arr if num > sum(arr)/len(arr)])}")